main.rb

```diff
@@ -17,7 +17,11 @@ def run
   @rental = []
   puts 'Welcome to School Library App!'
   puts "\n"
-  options
+  if File.exist?('./files/rentals.json') && File.exist?('./files/books.json') && File.exist?('./files/people.json')
+    options
+  else
+    puts 'pls check your file rentals.json or books.json or people.json'
+  end
```

Comment:
- [ ] Kindly note that the application should run and not prompt only a message when one of the JSON files is not present. Kindly create the correspondent file if is not present and then run the App.

![image](https://user-images.githubusercontent.com/65068771/158827625-29e2fedc-d144-4876-83c1-86b5df49ce46.png)